File: code/src/scripts/audits/fixtures/dict-discipline/broken/offenders.rs
//! Known-positive fixture — every line here MUST be reported. Never "fix" this file.

use std::collections::HashMap;

pub fn decode(raw: &str) -> HashMap<String, serde_json::Value> { // R1 + R2
    let _ = raw;
    HashMap::new()
}

pub struct Envelope {
    pub body: serde_json::Value, // R2
}

File: code/src/scripts/audits/fixtures/dict-discipline/broken/offenders.ts
// Known-positive fixture — every line here MUST be reported. Never "fix" this file.

export type Payload = Record<string, unknown>; // T1

export function parse(raw: string): Record<string, any> {
  // T1
  return JSON.parse(raw);
}

export interface Bag {
  [key: string]: string; // T2
}

export function accept(value: object): void {
  // T2
  void value;
}
